docs/Developers/Review.rst

```diff
@@ -72,6 +72,12 @@ The review process for coala is as follows:
     - ``reack commit_SHA`` in case the commit was acknowledged before, was
       rebased without conflicts and the rebase did not introduce logical
       problems.
+
+    .. note::
+
+        Only one acknowledge is needed per commit. There is no need to ack
```

Comment:
I would reword this to say `Only one acknowledgement is needed per commit i.e. ack <commit-sha>.`